Slider/tests/Slider-specs.js

```diff
@@ -279,7 +279,7 @@ describe('Slider', () => {
 		expect(actual).toBe(expected);
 	});
 
-	test('should not call onSpotlightLeft on horizontal slider at greater than min value', () => {
+	test.skip('should not call onSpotlightLeft on horizontal slider at greater than min value', () => {
```

Comment:
Should confirm with Blake but I believe we didn't pull this spotlight/5-way code over from Moonstone so these can probably be deleted.